src/app.js

```diff
@@ -1,42 +1,84 @@
 const { app } = require("./support/setupExpress");
-const { query } = require("./support/db");
+// const { query } = require("./support/db");
 const { gameOfThronesEpisodes } = require("./data/gameOfThronesData");
-
+const { filterThroughEpisodes } = require("./functions.js");
+const { generateEpisodeCode } = require("./functions.js");
+const { removePTags } = require("./functions.js");
 /** 
  @typedef {import('./data/episodeType').Episode} Episode
 */
 
 //You can delete this once you see the episodes have loaded ok.
-summariseEpisodesToConsole(gameOfThronesEpisodes);
+//summariseEpisodesToConsole(gameOfThronesEpisodes);
+
+//configure the server's route handlers here --
 
-//configure the server's route handlers
 app.get("/", (req, res) => {
     res.render("pages/index");
 });
 
-app.get("/db-test", async (req, res) => {
-    try {
-        const dbResult = await query("select now()");
-        const rows = dbResult.rows;
-        res.json(rows);
-    } catch (err) {
-        console.error(err);
-        res.status(500).send(
-            "Sorry, an error occurred on the server.  Ask the dev team to check the server logs at time " +
-                new Date(),
-        );
-    }
+app.get("/fullList", (req, res) => {
+    const wordToSearch = req.query.searchTerm;
+    const filteredEpisodes = filterThroughEpisodes(wordToSearch);
+    res.render("pages/fullList", {
+        gameOfThronesEpisodes,
+        episodeCodes: episodeCodes,
```

Comment:
you've used different naming convention on lines 25-28 instead of reducing to one word like line 24